tests/jvm/src/test/resources/example/NamedApplyBlock.scala

```diff
@@ -10,5 +10,5 @@ object NamedApplyBlockMethods/*<=example.NamedApplyBlockMethods.*/ {
 object NamedApplyBlockCaseClassConstruction/*<=example.NamedApplyBlockCaseClassConstruction.*/ {
   case class Msg/*<=example.NamedApplyBlockCaseClassConstruction.Msg#*/(body/*<=example.NamedApplyBlockCaseClassConstruction.Msg#body.*/: String/*=>scala.Predef.String#*/, head/*<=example.NamedApplyBlockCaseClassConstruction.Msg#head.*/: String/*=>scala.Predef.String#*/ = "default", tail/*<=example.NamedApplyBlockCaseClassConstruction.Msg#tail.*/: String/*=>scala.Predef.String#*/)
   val bodyText/*<=example.NamedApplyBlockCaseClassConstruction.bodyText.*/ = "body"
-  val msg/*<=example.NamedApplyBlockCaseClassConstruction.msg.*/ = Msg/*=>example.NamedApplyBlockCaseClassConstruction.Msg.*/(bodyText/*=>example.NamedApplyBlockCaseClassConstruction.bodyText.*/, tail/*=>example.NamedApplyBlockCaseClassConstruction.Msg.apply().(tail)*/ = "tail")
+  val msg/*<=example.NamedApplyBlockCaseClassConstruction.msg.*/ = /*=>example.NamedApplyBlockCaseClassConstruction.Msg.apply$default$2().*/Msg/*=>example.NamedApplyBlockCaseClassConstruction.Msg.*/(bodyText/*=>example.NamedApplyBlockCaseClassConstruction.bodyText.*/, tail/*=>example.NamedApplyBlockCaseClassConstruction.Msg.apply().(tail)*/ = "tail")
```

Comment:
Can you clarify why this occurrence is needed? This is a synthetic symbol

Reply:
It's not, I didn't noticed that it got added. I'm working  on removing it.